### PR TITLE
support .config directory for mill-version

### PR DESCRIPTION
**Pull request by nrktkt**
for parity with https://github.com/com-lihaoyi/mill/pull/945

### CODE REVIEW

File: millw.bat [hunk 1]
```diff
@@ -37,6 +37,9 @@ if [!MILL_VERSION!]==[] (
   if exist .mill-version (
       set /p MILL_VERSION=<.mill-version
   )
+  else if exist .config\mill-version (
```
Comment (nrktkt) [inline millw.bat:40]: been a long time since I wrote batch, check if this is correct

Comment (lefou) [reply millw.bat:40]: I think we need extra parenthesis in the else section, but have currently no system to test it.

**Review by lefou — COMMENTED** [review]
Thanks for this PR. I still need to make sure, the batch script is OK. I'm not a Windows user. Also, it would be nice to update the Readme about the `.config/mill-version` lookup.

**Comment by lefou** [comment]
Thank you!

**Pull request merged**